cmd/crc/cmd/start.go

```diff
@@ -320,6 +339,23 @@ func writePodmanTemplatedMessage(writer io.Writer, s *startResult) error {
 	})
 }
 
+func writeMicroShiftTemplatedMessage(writer io.Writer, s *startResult) error {
+	parsed, err := template.New("template").Parse(startTemplateForMicroshift())
+	if err != nil {
+		return err
+	}
+	userShell, err := shell.GetShell("")
+	if err != nil {
+		userShell = ""
+	}
+
+	return parsed.Execute(writer, &templateVariables{
+		EvalCommandLine:   shell.GenerateUsageHint(userShell, "crc oc-env"),
+		CommandLinePrefix: commandLinePrefix(userShell),
+		KubeConfigPath:    constants.KubeconfigFilePath,
+	})
+}
+
```

Comment:
These cmd/start.go changes could be split to their own small commit at the end of the PR

Reply:
In cmd/start.go only changes are related to microshift preset to generate the template, I am not sure if we split it into different commits because without `microshift` it doesn't have any usecase.